ios/Classes/MapEvents+JSON.swift

```diff
@@ -9,8 +9,8 @@ protocol MapEventEncodable {
 extension EventTimeInterval {
     var toJSON: [String: Any] {
         [
-            "begin": begin,
-            "end": end
+            "begin": begin.milisecondsSince1970,
```

Comment:
Flutter docs state microseconds for the timestamp, maybe we could try using DateTime instead? Seems that it's already confusing for us(milli/micro), probably will be more some to the users.